go.mod

```diff
@@ -123,6 +123,8 @@ require (
 replace (
 	github.com/gogo/protobuf => github.com/gogo/protobuf v1.3.2
 	github.com/gorilla/websocket => github.com/gorilla/websocket v1.4.2
+
+	k8s.io/api => k8s.io/api v0.26.2
```

Comment:
This looks unstable, that we are forcing older version of `k8s.io/api` than `kubevirt` expects. But it's ok for now, until controller-runtime is updated in a new PR.